keps/sig-node/3573-device-plugin/README.md

```diff
@@ -603,16 +688,10 @@ the Device Plugins.
 
 ### Version Skew Strategy
 
-Currently we require exact version match between Kubelet and Device Plugin.
-API version is expected to be increased only upon incompatible API changes.
-
-Follow protobuf guidelines on versioning:
-  * Do not change ordering
-  * Do not remove fields or change types
-  * Add optional fields
-  * Introducing new fields with proper default values
-  * Freeze the package name to `apis/device-plugin/v1alpha1`
-  * Have kubelet and the Device Plugin negotiate versions if we do break the API
+Prior to v1.10, the versioning scheme required the Device Plugin's API version to
+match exactly the Kubelet's version. With the graduation of this feature to Beta
+and move of device plugin API is to a stable API (version v1beta1) backward
```

Comment:
same comment as above "stable API" and "version v1beta1" seems a bit of a weird mix (and again: GA'ing the device plugin API is IMO out of scope of this KEP. Can't say if, per due process, is a prerequisite or can be done later. I just don't know).

Reply:
Will fix this as well. I am not sure as well if graduation of the feature implies we _have_ to move the API to GA or not. For now, I will assume that we are only graduating Device Manager to GA and not the Device Plugin API.

Reply:
I'm fine with this approach. Thanks!